non_lts_galactic/snap/snapcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: ros2-talker-listener
```

Comment:
Please add some comments to this file to (briefly) explain the different bits that are normally hidden by the extension.

Reply:
I added comments, let me know if it's fine for you